scripts/get-cdnjs-list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import json
+import subprocess
+
+links = {}
+data = subprocess.check_output(['links', '-dump', 'https://cdnjs.com/'])
```

Comment:
@ralsina, but there is no setting to tell Nikola what libraries to load. Each library is hardcoded in the theme and not loaded based on a configurable setting.